.github/styles/Vocab/Docs/accept.txt

```diff
@@ -115,4 +115,6 @@ wget
 Workbench
 Zabbix
 Zipkin
-ZooKeeper
\ No newline at end of file
+ZooKeeper
+redli
```

Comment:
The product name is "Redli" (with a capital initial letter). The command is lowercase but then should always use literal markup: ``redli``. Please be very careful to avoid adding case-insensitive entries for proper nouns to the spelling list; we should not use these words in this way.